tests/e2escenarios/e2e_source_test.go

```diff
@@ -86,6 +86,9 @@ var _ = Describe("odo source e2e tests", func() {
 	Context("odo component creation", func() {
 
 		It("Should be able to deploy a python source application", func() {
+			if runtime.GOARCH == "ppc64le" {
```

Comment:
Lets have only one Context for Skip. Could you please change the tests according to https://github.com/openshift/odo/pull/4000#discussion_r494016243